src/java.base/share/classes/sun/reflect/annotation/AnnotationSupport.java

```diff
@@ -77,7 +76,7 @@
             boolean indirectFirst = direct == null ||
                                     containerBeforeContainee(annotations, annoClass);
 
-            result.addAll((indirectFirst ? 0 : 1), Arrays.asList(indirect));
+            result.addAll((indirectFirst ? 0 : 1), List.of(indirect));
```

Comment:
This `indirect` is most likely to be of size > 2

Reply:
I've reverted this along with the rest of dubious changes